includes/PageVerificationRepo.php

```diff
@@ -6,6 +6,7 @@
 
 interface PageVerificationRepo {
 
-	public function getPageVerificationData( int $revId ): array;
+	// TODO: does this naming make sense? Which one is it: page or revision?
+	public function getPageVerificationData( int $revisionId ): array;
```

Comment:
Yes I agree it's clearer to be named `getRevisionVerificationData`.

Reply:
Same goes for the class name and the DB table I guess?

Reply:
Yes